lix/packages/client/src/lix/checkoutPlaceholders.ts

```diff
@@ -0,0 +1,86 @@
+import type { RepoState, RepoContext } from "../openRepository.js"
+import isoGit from "../../vendored/isomorphic-git/index.js"
+import { doCheckout } from "../git/checkout.js"
+import { modeToFileType } from "../git/helpers.js"
+
+export async function checkOutPlaceholders(ctx: RepoContext, state: RepoState) {
+	const { rawFs, cache, dir, experimentalFeatures } = ctx
+	const { branchName, checkedOut, sparseFilter } = state
+
+	if (!experimentalFeatures.lazyClone) {
+		return
```

Comment:
I would rather throw in such a case - this function should not be called when the flag is not turned on?

Reply:
i think we dont have to handle this at all anyomore.